scripts/tests/CRM/test_1b_decrypt_fidelity.py

```
@@ -63,31 +63,70 @@ def inspect_raw_database(db_path: str, user_id: int) -> dict:
+        "name", "encrypted_name", "ephemeral_pubkey_name", "email_blind_index",
+        "user_type_id", "approved", "created_at"
+    ]
+    values = lines[0].split("|")
+    if len(values) != len(columns):
+        return None
 
-    # Get field values
-    cmd = f"docker compose exec -T backend sqlite3 -json {shlex.quote(db_path)} 'SELECT fd.field_name, ufv.value, ufv.encrypted_value, ufv.ephemeral_pubkey FROM user_field_values ufv JOIN user_field_definitions fd ON fd.id = ufv.field_id WHERE ufv.user_id = {user_id}'"
-    result = subprocess.run(cmd, shell=True, capture_output=True, text=True, cwd=repo_root)
+    user_data = {}
+    for col, val in zip(columns, values):
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)